api/ChannelSplitterNode.json

```diff
@@ -0,0 +1,107 @@
+{
+  "api": {
+    "ChannelSplitterNode": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/ChannelSplitterNode",
+        "support": {
+          "webview_android": {
+            "version_added": true
+          },
+          "chrome": {
+            "version_added": "14"
+          },
+          "chrome_android": {
+            "version_added": "14"
+          },
+          "edge": {
+            "version_added": true
+          },
+          "edge_mobile": {
+            "version_added": true
+          },
+          "firefox": {
+            "version_added": "25"
```

Comment:
Has similar changes to the one of Chrome already happened in Firefox? If so -> note.

Reply:
No, I don't think so.